build.gradle

```diff
@@ -99,8 +99,8 @@ subprojects { subproject ->
 		log4j2Version = '2.7'
 		logbackVersion = '1.1.7'
 		mockitoVersion = '1.10.19'
-		rabbitmqVersion = project.hasProperty('rabbitmqVersion') ? project.rabbitmqVersion : '3.6.5'
-		rabbitmqHttpClientVersion = '1.0.0.RELEASE'
+		rabbitmqVersion = project.hasProperty('rabbitmqVersion') ? project.rabbitmqVersion : '4.0.0'
```

Comment:
Although the version number looks like a major release; I don't believe it's much different to the 3.6.x client - the numbering reflects that it has been detached from server versioning.

http://www.rabbitmq.com/blog/2016/11/24/rabbitmq-java-client-4-0-is-released/